src/System.Drawing.Common/src/System/Drawing/Pens.cs

```diff
@@ -152,2542 +148,177 @@ public sealed class Pens
         private static readonly object s_yellowKey = new object();
         private static readonly object s_yellowGreenKey = new object();
```

Comment:
Same lazy init feedback as in the previous case.  Plus, seeing that this is repeated multiple times, in a subsequent PR we should be able to consolidate all of these to just have a single set of fields which are then used by each of Pens, Brushes, Colors, etc.  Doing that would likely changing how ThreadData works, e.g. by switching each of the classes to have its own [ThreadStatic] dictionary, but that's better anyway.